files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```diff
@@ -39,6 +39,14 @@ tools:
     env:
       TEMP: /data/1/galaxy_db/tmp
 
+  gmx_sim:
+    gpus: 1
```

Comment:
@mira-miracoli @sanjaysrikakulam @cat-bro Is the setting `resources: usegpu` in job_conf.xml equivalent to `gpus: 1` in tools.yml?

Reply:
@mira-miracoli @sanjaysrikakulam According to @cat-bro, `gpus: 1` in TPV means "the destination has at least 1 gpu", so you would just need to confirm whether `resources: usegpu` has exactly the same meaning.

Reply:
IIRC: the following should be added 

```
scheduling:
      require:
        - condor-gpu
```

Reply:
yes we discussed this, because our experience was that gpu destinations also take jobs that don't specify GPUs, which we would like to avoid. (or did I get something wrong @cat-bro)

Reply:
I set up the rule-based solution in [6808149](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/689/commits/68081490337205a94ea641a9b1fcba1f3366d27a), closing the conversation.